src/components/largePipelineWarning/largePipelineWarning.scss

```diff
@@ -0,0 +1,36 @@
+@import '../../styles/variables';
+
+.pipeline-warning {
+  position: absolute;
+  top: 30%;
+  left: 50%;
+  color: white;
```

Comment:
This will need to take on the standard theming approach, as at the moment this only works for the dark theme, you can refer to how it's done in other components but also check with @richardwestenra on this.

Reply:
Nice suggestion! I have modified the CSS according to the selected theme